aleph-client/src/debug/mod.rs

```diff
@@ -0,0 +1,23 @@
+use crate::Connection;
+
+mod aleph;
+mod elections;
+mod treasury;
+
+fn pallet_prompt(name: &'static str) -> String {
+    format!("-----------{}-----------", name)
+}
+
+fn entry_prompt(name: &'static str) -> String {
+    format!("----{}", name)
+}
+
+fn element_prompt(out: String) -> String {
+    format!("\t{}", out)
+}
+
+pub fn print_storages(connection: &Connection) {
+    treasury::print_storage(&connection);
```

Comment:
We don't need `&` here as `connection` is already a ref?

Reply:
you are right, done